### PR TITLE
Remove show from trakt watchlist that are not on tvdb

### PR DESCRIPTION
**Pull request by gborri**

### CODE REVIEW

**Comment by gborri** [comment]
related to SiCKRAGETVsickrage-issues/#1959

**Comment by gborri** [comment]
i have no idea how to suppress traceback message on logfile. can anyone could help?

**Comment by miigotu** [comment]
What traceback?

File: sickbeard/show_queue.py [hunk 1]
```diff
@@ -282,12 +283,35 @@ def run(self):
                 self._finishEarly()
                 return
         except Exception, e:
-            logger.log(u"Show name with ID %s don't exist in %s anymore. Please change/delete your local .nfo file or remove it from your TRAKT watchlist" %
+            logger.log(u"Show name with ID %s don't exist in %s anymore. it will be removed form your TRAKT watchlist" %
                 (self.indexer_id,sickbeard.indexerApi(self.indexer).name) , logger.ERROR)
```
Comment (miigotu) [inline sickbeard/show_queue.py]: `s/form/from/`
also, please change it from `don't exist in` to `don't exist on` for proper english xD.

**Pull request closed without merging**